packages/suite/src/components/guide/GuideHeaderBreadcrumb.tsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import styled from 'styled-components';
+import { darken } from 'polished';
+
+import { GuideCategoryType } from '@suite-types/guide';
+import { useActions, useSelector } from '@suite/hooks/suite';
+import * as guideActions from '@suite-actions/guideActions';
+import { variables } from '@trezor/components';
+import { Translation } from '@suite-components';
+import { findParentCategories, getItemTitle } from '@suite-utils/guide';
+
+const BreadcrumbWrapper = styled.span`
+    font-size: ${variables.FONT_SIZE.SMALL};
+    font-weight: ${variables.FONT_WEIGHT.MEDIUM};
+`;
+
+const PreviousCategoryLink = styled.a`
+    color: ${props => props.theme.TYPE_LIGHT_GREY};
+
+    &:hover {
+        color: ${props => darken(props.theme.HOVER_DARKEN_FILTER, props.theme.TYPE_LIGHT_GREY)};
+    }
+`;
+
+const BreadcrumbDelimiter = styled.span`
+    color: ${props => props.theme.TYPE_LIGHT_GREY};
+    margin: 0 5px;
+`;
+
+const CategoryLink = styled.a`
+    color: ${props => props.theme.TYPE_GREEN};
+
+    &:hover {
+        color: ${props => darken(props.theme.HOVER_DARKEN_FILTER, props.theme.TYPE_GREEN)};
+    }
+`;
+
+const GuideHeaderBreadcrumb = () => {
+    const { language, item, root } = useSelector(state => ({
+        item: state.guide.item,
+        language: state.suite.settings.language,
+        root: state.guide.root,
+    }));
+
+    const { setView, openItem } = useActions({
+        setView: guideActions.setView,
+        openItem: guideActions.openItem,
+    });
+
+    const fallbackBreadcrumb = (
+        <BreadcrumbWrapper>
+            <CategoryLink onClick={() => setView('GUIDE_DEFAULT')}>
+                <Translation id="TR_GUIDE_DASHBOARD" />
+            </CategoryLink>
+        </BreadcrumbWrapper>
+    );
+
+    if (!item || !root) return fallbackBreadcrumb;
+
+    const parentCategories = findParentCategories(item, root);
+
+    if (!parentCategories?.length) return fallbackBreadcrumb;
+
+    const parentCategory = parentCategories[parentCategories.length - 1];
+    const parentCategoryTitle = getItemTitle(parentCategory, language);
+
+    let grandParentCategory: GuideCategoryType;
+    let grandParentCategoryTitle;
+    if (parentCategories.length > 1) {
+        grandParentCategory = parentCategories[parentCategories.length - 2];
+        grandParentCategoryTitle = getItemTitle(grandParentCategory, language);
+    }
```

Comment:
I guess this is meant to be generalized later, right?

Anyway, one could make this a bit nicer with something like
```suggestion
    const parent = parentCategories.pop();
    const parentTitle = parent && getItemTitle(parent, language);
    const grandParent = parentCategories.pop();
    const grandParentTitle = grandParent && getItemTitle(grandParent, language);
```

Reply:
> If you call pop() on an empty array, it returns undefined.

Nice, good point

Reply:
Resolved.